src/main/java/knight2103/command/InstructionInvalid.java

```diff
@@ -0,0 +1,4 @@
+package knight2103.command;
+
+public class InstructionInvalid extends Exception {
```

Comment:
Perhaps can be InstructionInvalidException instead

Reply:
Ah okay thanks!
I don't know where to write the reply but thanks for your main comment:
> Main comments your Deadline, Event Todo tasks can all be renamed to DeadlineTask, EventTask and ToDoTask so that it is clearer that they inherit from Task

It's a good suggestion.